packages/loot-core/src/server/accounts/parse-file.test.ts

```diff
@@ -84,18 +84,6 @@ describe('File import', () => {
     expect(await getTransactions('one')).toMatchSnapshot();
   }, 45000);
 
-  test('ofx import works with multiple decimals in amount', async () => {
```

Comment:
This test doesn't mean anything anymore since the file parser was returning a string for the value which would get converted later